docs/usage/configuration-options.md

```diff
@@ -2659,15 +2659,21 @@ Avoid setting `rebaseWhen=never` and then also setting `prCreation=not-pending`
 
 ## recreateClosed
 
-By default, Renovate will detect if it has proposed an update to a project before and not propose the same one again.
-For example the Webpack 3.x case described above.
-This field lets you customize this behavior down to a per-package level.
-For example we override it to `true` in the following cases where branch names and PR titles need to be reused:
+By default, Renovate detects if it proposed an update to a project before, and not propose the same one again.
+For example the Webpack 3.x case described in the `separateMajorMinor` documentation.
+You can use `recreateClosed` to customize this behavior down to a per-package level.
+For example we override it to `always` in the following cases where branch names and PR titles need to be reused:
```

Comment:
I'm not sure what `we override it to ...` refers to:

- we the maintainers override to `always` in our own personal Renovate bot config? 
- we the maintainers hard-coded some overrides into the Renovate bot code? (basically the bot ignores the presets/user config and always does `x`)?

Reply:
It means that if you use the options `auto | always` then still for immortal PRs the options we internally apply is `always`.

This sentence needs to change though, because now we have a third option `never` which doesn't let us apply `always` to immortal PRs